app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -169,6 +170,7 @@ const AppLayoutWithExperiment = ({
             computedEndDate,
             status,
             computedDurationDays,
+            isArchived,
```

Comment:
Feels like we're getting pretty close to just passing the whole experiment down lol

Reply:
Oh yeah I realized this after I wrote it.  Like I could put `isArchived` into the little handy dandy `status` thingy, but then whenever we want some other random bool we gotta stick it in there?  I like just passing the whole experiment around, I mean why not?  Also I'm not exactly sure how we have the hooks setup, but in theory can't some random component just use `useExperiment` directly without needing it from the parent?  Or does invoking the hook make the fetch all?

Reply:
Yeah useExperiment will make the fetch call. But soon, once we move to Context, we'll be able to pull experiment data from the context and not need to pass the experiment object or individual properties down as props :)